fvm/transactionInvoker.go

```diff
@@ -87,6 +89,15 @@ func newTransactionExecutor(
 		ctx.EnvironmentParams,
 		txnState)
 
+	cadenceRuntime := env.BorrowCadenceRuntime()
+
+	// TODO: how to clean up?
```

Comment:
That's a good question! So far there's no support for "undeclaring". Maybe open an issue for that in the Cadence repo and we'll have to see if/how that's possible